client/src/components/DataFiles/DataFiles.js

```diff
@@ -113,11 +113,9 @@ const DataFiles = () => {
   const loading = useSelector(state => state.systems.storage.loading);
   const error = useSelector(state => state.systems.storage.error);
   const systems = useSelector(state => state.systems.storage.configuration);
-
```

Comment:
Do we want these spacing's removed?  Maybe, an artifact of looking for the issue?